### DISCUSSION TITLE
remote PC run

### DISCUSSION THREAD

**Discussion by wincmd64** [fastfetch-cli/fastfetch#1126] [Q&A]
How can I run a fastfetch on a remote PC using custom config.jsonc?
(Windows 10)

**Comment by CarterLi** [comment]
Elaborate please

**Reply by FelixFourcolor** [reply]
I don't understand the question.

```
ssh username@server
fastfetch
```

Or do you mean something else?

**Reply by hasecilu** [reply]
> I don't understand the question.
> 
> ```
> ssh username@server
> fastfetch
> ```
> 
> Or do you mean something else?

I was thinking the same. I tried `ssh user@home-server "fastfetch -l debian"` but only the data is shown, logo is missing. When inside of the home server ascii logo is shown as expected.
